### PR TITLE
Swift objects info

### LINKED ISSUE #93: swift info module
Need to add swift info for containers, container objects, object content

### PR DESCRIPTION
**Pull request by anton-sidelnikov**
Close #93

### CODE REVIEW

**Review by gtema — DISMISSED** [review]
We must do some changes:
- module name is wrong, it should be "object_info"
- actually this should go into openstack collection, but it's ok to have it here first and backport it
- potentially we need instead modify "object" to allow object downloading to be similar to https://github.com/ansible-collections/google.cloud/blob/master/plugins/modules/gcp_storage_object.py and https://github.com/ansible-collections/amazon.aws/blob/main/plugins/modules/aws_s3.py. _info module in this case should verify presence of an object and return it's metadata/size/etc

**Comment by anton-sidelnikov** [comment]
> We must do some changes:
> 
> * module name is wrong, it should be "object_info"
> * actually this should go into openstack collection, but it's ok to have it here first and backport it
> * potentially we need instead modify "object" to allow object downloading to be similar to https://github.com/ansible-collections/google.cloud/blob/master/plugins/modules/gcp_storage_object.py and https://github.com/ansible-collections/amazon.aws/blob/main/plugins/modules/aws_s3.py. _info module in this case should verify presence of an object and return it's metadata/size/etc

Ok, downloading of objects will be in another module. Here only getting of containers, and objects metadata.

File: plugins/modules/object_info.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+        "range": null,
+        "signature": null,
+        "timestamp": "1613655609.64076",
+        "transfer_encoding": null
+      }
+'''
+
+EXAMPLES = '''
+# Get swift containers/objects/object content.
+- object_info:
```
Comment (Polina-Gubina) [inline plugins/modules/object_info.py:139]: maybe full path like 'opentelekomcloud.cloud.object_info' ?

**Review by Polina-Gubina — APPROVED** [review]

File: plugins/modules/object_info.py (hunk 1 above)
Comment (anton-sidelnikov) [reply plugins/modules/object_info.py]: yes, sure

File: plugins/modules/object_info.py [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+RETURN = '''
+swift:
+  description: List of dictionaries describing containers, objects
+    and object metadata in container.
+  type: complex
+  returned: On Success.
+  contains:
+    containers:
+      description: Specifies the lost of available containers.
```
Comment (Polina-Gubina) [inline plugins/modules/object_info.py:42]: a misprint, list

**Comment by anton-sidelnikov** [comment]
recheck

**Review by tischrei — APPROVED** [review]

**Pull request merged**